generator/html/jbuild

```diff
@@ -0,0 +1,10 @@
+(jbuild_version 1)
+
+(executable (
+  (name www)
+  (libraries (
+    generator_lib
+    cow
+    caml2html
```

Comment:
Not a problem with jbuilder, but I think this is missing from `xs-opam`

Reply:
I think it may not be an issuse because the spec file does not seem to call the html target; but perhaps we may want to add it?

Reply:
I don't think travis calls that either.

Reply:
would be nice to automatically regenerate the docs, like for xapi, but that can be done as separate PR.

Reply:
With the above changes, it is a matter of simply calling `make html` - but we'll need caml2html for this.

Reply:
Let's add it to xs-opam :)

Reply:
As discussed, this will be a diffrent PR.